src/renderer/entities/wallet/ui/index.ts

```diff
@@ -3,4 +3,5 @@ export { AccountsList } from './AccountsList/AccountsList';
 export { AddressWithName } from './AddressWithName/AddressWithName';
 export { AccountAddress, getAddress } from './AccountAddress/AccountAddress';
 export { AddressWithTwoLines } from './AddressWithTwoLines/AddressWithTwoLines';
+export { WalletIcon } from './WalletIcon/WalletIcon';
```

Comment:
I assume this icons must be used only through `<WalletIcon />` component?

Reply:
Yeah, we doing same thing for assets with AssetIcon component and I think it's proved to be convinient